admin_guide/persistent_storage/pod_security_context.adoc

```diff
@@ -0,0 +1,243 @@
+= Pod Security Context: Volume security
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+This document provides a general guide on pod security context as it
+relates to volume security. For information on pod security context in
+general please refer to link:../manage_scc.html[Managing Security
+Context Constrains] documentation.
+
+The Pod Security Context provides three fields which allow you to gain
+and control access to volumes. These are
+`FSGroup`,`SupplementalGroups`, and `SELinuxOptions`.
+
+[IMPORTANT]
+====
+It is important to note that `FSGroup` and `SELinuxOptions` are
+generated for your pod if not specified. It is recommended that you
+rely on the automatically generated values and not specify your own
+unless you have a specific use case which requires that.
+====
+
+== SupplementalGroups
+
+Supplemental groups are regular Linux groups. When a process run in
+Linux it has UID and one or more GID's. Usually this is just the UID,
+and GID's of user who started the process, but in the case of
+containers you have the option of settings these attributes for the
+container's main process. The `SupplementalGroups` option of the pod
+`SecurityContext` takes advantage of that and enables you to specify a
+list of groups which will be added to the main process of every
+container in the pod.
+
+This is useful because it enables you to give group based access to
+volumes to your pods.
```

Comment:
> access of volumes to

"Access of volumes" does not sound quite right to me, but I have rewritten the sentence to avoid the awkward structure above. PTAL